src/libraries/System.Formats.Cbor/src/System/Formats/Cbor/Writer/CborWriter.Tag.cs

```diff
@@ -79,11 +79,11 @@ public void WriteUnixTimeSeconds(double seconds)
 
         /// <summary>Writes the provided value as a tagged bignum encoding, as described in RFC7049 section 2.4.2.</summary>
         /// <param name="value">The value to write.</param>
-        /// <exception cref="InvalidOperationException">Writing a new value exceeds the definite length of the parent data item.
-        /// -or-
-        /// The major type of the encoded value is not permitted in the parent data item.
-        /// -or-
-        /// The written data is not accepted under the current conformance mode.</exception>
+        /// <exception cref="InvalidOperationException"><p>Writing a new value exceeds the definite length of the parent data item.</p>
+        /// <p>-or-</p>
+        /// <p>The major type of the encoded value is not permitted in the parent data item.</p>
+        /// <p>-or-</p>
+        /// <p>The written data is not accepted under the current conformance mode.</p></exception>
```

Comment:
Personally, I don't like this.  From an XMLdoc perspective the contents of `<exception>` is just text (at least, that's the impression that I get from https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments#d35-exception).  Using `<p>` is embedding the knowledge that we're consuming this into HTML; it wouldn't be correct for a Unix manpage, or markdown, or anything other than HTML.

It feels like either we should CDATA the contents to try to preserve the newlines, or change our doc style/generator process to be something like

```suggestion
        /// <exception cref="InvalidOperationException">Writing a new value exceeds the definite length of the parent data item.</exception>
        /// <exception cref="InvalidOperationException">The major type of the encoded value is not permitted in the parent data item.</exception>
        /// <exception cref="InvalidOperationException">The written data is not accepted under the current conformance mode.</exception>
```

and then the xmldoc -> website generator glues the three of them together with paragraphs and `-or-`

Reply:
I think we should either use CDATA, or another thought is to use a bulleted list, e.g.

```
/// <exception cref="T:System.Formats.Cbor.CborContentException">
/// <list type="bullet">
/// <item><description>The next value has an invalid CBOR encoding, or</description></item>
/// <item><description>There was an unexpected end of CBOR encoding data, or</description></item>
/// <item><description>The next value uses an encoding that's not valid under the current conformance mode.</description></item>
/// </list>
/// </exception>
```

Reply:
A list seems reasonable.  It has the advantage of getting rid of the `-or-` paragraphs, so there's no `-or-` vs `--or--` vs `- or -` (etc).

It also looks like the XMLdoc syntax has `<para>` (vs HTML `<p>`).  So I guess I'd also be OK with `<para>` (which we use in `<remarks>` sections already)

Reply:
I've changed them all to `<para>`.